azurerm/internal/services/policy/resourceids.go

```diff
@@ -0,0 +1,3 @@
+package policy
+
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=GuestConfigurationAssignment -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Compute/virtualMachines/vm1/providers/Microsoft.GuestConfiguration/guestConfigurationAssignments/assignment1 -rewrite=true
```

Comment:
could we update this to reflet the new name?

Reply:
No problem. One side effect of this will be that one of the property of the ID struct will change from straightforward `Name`  to `GuestConfigurationName`